data/registry/instrumentation-dotnet-nservicebus.yml

```
@@ -0,0 +1,24 @@
+# cSpell:ignore # add words unknown to cspell, remove this line if not required
+title: NServiceBus OpenTelemetry Integration # the name of your project
+registryType: instrumentation # the type of integration; is this an exporter, plugin, API package, or something else?
+language: dotnet
+tags:
+  - nservicebus
+  - dotnet
+urls:
+    website: https://particular.net/ # to work on the integrations page as well
+    repo: https://github.com/Particular/NServiceBus
+    docs: https://docs.particular.net/nservicebus/operations/opentelemetry
+license: Apache 2.0 # or whatever your OSS license is (see https://opensource.org/licenses/), proprietary or non-OSI approved may be rejected
+description: A friendly description of your integration/plugin
```

```suggestion
description: A friendly description of your integration/plugin
```

please provide a description

---

```suggestion
description: With NServiceBus OpenTelemetry instrumentation enabled, tracing, metrics, and logging can be individually configured via the OpenTelemetry API itself.
```